vpr/src/route/annotate_routing.cpp

```diff
@@ -47,11 +49,18 @@ vtr::vector<RRNodeId, ClusterNetId> annotate_rr_node_nets(const ClusteringContex
                  * In some routing architectures, node capacity is more than 1
                  * which allows a node to be mapped by multiple nets
                  * Therefore, the sanity check should focus on the nodes
-                 * whose capacity is 1
+                 * whose capacity is 1.
+                 * Flat routing may create two clustered nets from a single
```

Comment:
Might want to say why / what makes two clustered nets (split a single atom net onto two cluster block outputs)